pyintegration/tests/test_dtypes.py

```diff
@@ -0,0 +1,194 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+import math
+import time
+import unittest
+
+import jpy
+import numpy
+import numpy as np
+import pandas as pd
+
+from deephaven2 import dtypes
+from deephaven2.dtypes import DTypes
+from deephaven2.constants import *
+from tests.testbase import BaseTestCase
+
+
+class DTypesTestCase(BaseTestCase):
+    def test_type_alias(self):
+        self.assertEqual(DTypes.byte, DTypes.int8)
+        self.assertEqual(DTypes.short, DTypes.int16)
+        self.assertEqual(DTypes.int_, DTypes.int32)
+        self.assertEqual(DTypes.long, DTypes.int64)
+        self.assertEqual(DTypes.float_, DTypes.single)
+        self.assertEqual(DTypes.float_, DTypes.float32)
+        self.assertEqual(DTypes.double, DTypes.float64)
+
+    def test_j_type(self):
+        self.assertEqual(dtypes.bool_.j_type, jpy.get_type("java.lang.Boolean"))
+        self.assertEqual(dtypes.byte.j_type, jpy.get_type("byte"))
+        self.assertEqual(dtypes.short.j_type, jpy.get_type("short"))
+        self.assertEqual(dtypes.char.j_type, jpy.get_type("char"))
+        self.assertEqual(dtypes.int_.j_type, jpy.get_type("int"))
+        self.assertEqual(dtypes.long.j_type, jpy.get_type("long"))
+        self.assertEqual(dtypes.float_.j_type, jpy.get_type("float"))
+        self.assertEqual(dtypes.double.j_type, jpy.get_type("double"))
+        self.assertEqual(dtypes.string.j_type, jpy.get_type("java.lang.String"))
+        self.assertEqual(dtypes.BigDecimal.j_type, jpy.get_type("java.math.BigDecimal"))
+
+    def test_qst_type(self):
+        self.assertIn(".qst.type.", str(dtypes.short.qst_type))
```

Comment:
Should we be testing all types here?
Should the test compare the exact value, instead of simply being in qst?

Reply:
I believe we should remove this test. It is brittle because it has a strong coupling with the Java namespace and it doesn't actually test anything more than the initialization of these DType instances.